packages/session-replay-browser/src/session-replay.ts

```diff
@@ -103,6 +104,16 @@ export class SessionReplay implements AmplitudeSessionReplay {
     this.recordEvents();
   }
 
+  getSessionReplayDebugPropertyValue() {
+    let apiKeyHash = '';
+    if (this.config) {
+      apiKeyHash = generateHashCode(this.config.apiKey).toString();
```

Comment:
QQ: Is it necessary to convert it to a string value?

Reply:
I think the hash can be negative

Reply:
the hash code returns an int, a string value is more generic.